pom.xml

```diff
@@ -116,34 +116,34 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-job</artifactId>
-      <scope>test</scope>
+      <scope>compile</scope>
```

Comment:
You can remove scope instead of changing it.
Goes for all the once you marked compile